workflows/sesv2_weekly_mailer/content/.aillyrc

```diff
@@ -0,0 +1,53 @@
+---
+combined: true
+overwrite: false
+---
+
+TCX SDK Code Examples
```

Comment:
This should be a template now.

Reply:
Agreed, but 1.2.6 is still RC and I don't want to change it until after the workshop.

Reply:
Oh not necessarily, 1.2.5 has plugin `view`. I could do it there?